### PR TITLE
Heavy Armor Buff + Armor Linking Reactivation

### PR DESCRIPTION
**Pull request by Geevies**
## About The Pull Request

Enabled the armor linking system, meaning combat gloves and combat boots now benefit from wearing heavier armor. Additionally, heavy armor now has an integrated drop pouch.

## Why It's Good For The Game

Heavy armor is weak, because the increased armor only slowed you down and made you an easier target for foot and hand slashers. Now, with the hands and feet being protected more, as well as the increase in storage capacity (and potential storage items), it seems a little more worth taking now.

## Changelog

:cl:
balance: Enabled the armor linking system, meaning combat gloves and boots now benefit from wearing heavier armor.
balance: Gave Heavy armor an integrated drop pouch, which can store more and larger items.
/:cl:

### CODE REVIEW

**Comment by FlamermanGrills** [comment]
Lets fucking gooooooooo!!!

**Comment by JosephBiell** [comment]
Finally a change that is not only needed , but also deserved for marines.

**Comment by Geevies** [comment]
to be clear, armor linking works on all armors, the glove picks whatever value is higher, it doesn't just work on heavy armor

**Comment by c4xmaniac** [comment]
HOLY FUCK THE HEAVY ARMOR RENNAISANCE

**Comment by memesky117** [comment]
🧎‍♂️

**Comment by Erawh** [comment]
Does this also mean light armor gets a buff since the gloves are linked, or will the frac chance remain about the same as currently?

**Comment by PhantomEpicness** [comment]
Zased

**Comment by Carnage068** [comment]
YES!!!!

**Comment by usnpeepoo** [comment]
bring back heavy armor pounce blocking

**Comment by Blithering** [comment]
Amazed how many years targeting hands and feet has rendered armour choice irrelevant up to now.

**Comment by nuklearcellphoneg** [comment]
If you're going to integrate drop pouch to it, do it to all armor

**Comment by PhantomEpicness** [comment]
UNFATHOMABLY BASED 

_Disclaimer_

- CM-13 is not related to the Disney owned "Aliens" Franchise and IP This is an entirely fan-made project. In case of an investigation by any federal entity or similar, I do not have any involvement with this group or with the people in it, I do not know how I am here, probably added by a thrid party, I do not support any actions by the member of this group.

**Comment by Wintermote** [comment]
Don't see this changing anything.

The main killer in this game is stuns, not direct damage. This PR doesn't fix the fundamental problem with heavy armor: It doesn't actually make you more survivable. With the slowdown it actually does the opposite by making it harder to avoid stuns, and once you're stunned you're going to be killed no differently than if wore light armor.

Delimbs are also relatively rare and avoidable so long as you aren't fighting with high damage on limbs. So I wouldn't personally take heavy armor just to mitigate that when the cost for it is speed - which matters a great deal both defensively AND offensively.

If you really want to see people use heavy armor, have it apply a stun reduction so you get up faster from a pounce, shotgun blast, grenade explosion, etc. This is what would ACTUALLY improve survivability for the wearer - not damage resistance.

**Comment by Carnage068** [comment]
> Don't see this changing anything.
> 
> The main killer in this game is stuns, not direct damage. This PR doesn't fix the fundamental problem with heavy armor: It doesn't actually make you more survivable. With the slowdown it actually does the opposite by making it harder to avoid stuns, and once you're stunned you're going to be killed no differently than if wore light armor.
> 
> Delimbs are also relatively rare and avoidable so long as you aren't fighting with high damage on limbs. So I wouldn't personally take heavy armor just to mitigate that when the cost for it is speed - which matters a great deal both defensively AND offensively.
> 
> If you really want to see people use heavy armor, have it apply a stun reduction so you get up faster from a pounce, shotgun blast, grenade explosion, etc. This is what would ACTUALLY improve survivability for the wearer - not damage resistance.

Heavy Armor is supposed to make you tankier. That's its primary purpose. That's why it's called armor in the first place. It protects you- Except it doesn't, because people ignore 100% of your armor by slashing your feet, so the one thing you're supposed to have over Light Armor literally doesn't fucking exist. I don't really see how you don't see a problem with this. It's pretty easy to tell for me. If having the armor cover feet literally wouldn't change anything at all, then what's the problem with adding it? lol

**Comment by ApplePlastic** [comment]
Considering xenomorphs (runners and lurker mains) target feet instead of inner limbs, this is a good PR that better balances the drawbacks with something more reasonable.

**Comment by PhantomEpicness** [comment]
Down with the foot fetishists!

speedmerge NOW!

**Comment by AGasCanister** [comment]
E'ing an item into the heavy armor puts it under your skin (Grim knows more about this ask him)

**Comment by Ticker175** [comment]
Whilst this is great because it means I can now take 9 shotguns into battle!!!, I still will pick light armour because all this new heavy armour does is add a drop pouch at the cost of being more vulnerable to getting capped due to the fact that you are less mobile and have more core defence

**Comment by Wintermote** [comment]
> Heavy Armor is supposed to make you tankier. That's its primary purpose. That's why it's called armor in the first place. It protects you- Except it doesn't, because people ignore 100% of your armor by slashing your feet, so the one thing you're supposed to have over Light Armor literally doesn't fucking exist. I don't really see how you don't see a problem with this. It's pretty easy to tell for me. If having the armor cover feet literally wouldn't change anything at all, then what's the problem with adding it? lol

I wasn't arguing against the PR, just stating that this isn't going to change anything.

Nobody is wearing heavy armor to save them from slashed limbs. That isn't going to change even with this PR. People who are going to wear heavy armor want it to make them more survivable, and due to the way combat works in this game and quite contrary to your belief, the way to survive damage is to mitigate the amount of time you are stunned and immobile and getting slashed for free by something - and the armor that works best for this is LIGHT armor, not heavy.

If you introduce a reduction on stun length, that indirectly reduces the time a xeno has to get free hits on you and THAT increases your survivability against xenos, not damage reduction on hits.

I get the idea: Light armor, you move and dodge attacks or get out of bad situations quickly, and that's how you survive.  Heavy armor - you take the hits and shrug them off and survive.

But Heavy armor /does not work like that/; you take an extra - what, 2 slashes? 3? - you're more likely to stay in crit and be capped, you have nothing that stops you from being stunned the same as someone in light armor and you will die just as much as someone stunned in light armor will, which is going to be 90% of the time what happens when a xeno kills you, and to top it off you ALSO have the downside of being unable to chase and kill anything.

That is why I suggested - stun reduction on heavy armor to make people more survivable. You get up from a stun faster, that is less damage dealt to you, and done without turning you into a bulletsponge like higher damage resistance would.

**Comment by IngloriouslyBad** [comment]
> I wasn't arguing against the PR, just stating that this isn't going to change anything.
> 
> Nobody is wearing heavy armor to save them from slashed limbs. That isn't going to change even with this PR. People who are going to wear heavy armor want it to make them more survivable, and due to the way combat works in this game the way to survive damage is to mitigate the amount of time you are stunned and immobile and getting slashed for free by something - and the armor that works best for this is LIGHT armor, not heavy.
> 
> If you introduce a reduction on stun length, that indirectly reduces the time a xeno has to get free hits on you and THAT increases your survivability against xenos, not damage reduction on hits, whether its hands, feet, or whatever.
> 
> I get the idea: Light armor, you move and dodge attacks or get out of bad situations quickly, and that's how you survive. Heavy armor - you take the hits and shrug them off and survive.
> 
> But Heavy armor /does not work like that/; you take an extra - what, 2 slashes? 3? - you're more likely to stay in crit and be capped, you have nothing that stops you from being stunned the same as someone in light armor and you will die just as much as someone stunned in light armor will, which is going to be 90% of the time what happens when a xeno kills you, and to top it off you ALSO have the downside of being unable to chase and kill anything.
> 
> That is why I suggested - stun reduction on heavy armor to make people more survivable. You get up from a stun faster, that is less damage dealt to you, and done without turning you into a bulletsponge like higher damage resistance would.


Exactly my thoughts. This will pretty much make more people curious into taking the trap that is heavy armor. More people will be put into crit instead of dying outright, and we're going to see more captures. More xenos will escape due to heavy armor being so restrictive in terms of mobility, chasing... and ... well ... everything.

But hey, your limb might not break after 2 hits, and you get a little bit of extra inventory! womp womp...

**Comment by c4xmaniac** [comment]
After playing 4 or 5 rounds with it I got to say, awesome change. Would be cool if you also implemented it for the riot armor found on the maps, for the past 6 years any time you've grabbed the armor as a surv people would leave you to die. Finally this is the MOMENT to redeem all the 5 hour survivor players AGONY by making riot armor VIABLE AFTER 6 YEARS!!!!!!!!

File: code/modules/clothing/shoes/marine_shoes.dm [hunk 1]
```diff
@@ -21,6 +21,10 @@
 	var/armor_stage = 0
 	items_allowed = list(/obj/item/attachable/bayonet, /obj/item/weapon/melee/throwing_knife, /obj/item/weapon/gun/pistol/holdout, /obj/item/weapon/gun/pistol/m43pistol, /obj/item/tool/screwdriver)
 
+/obj/item/clothing/shoes/marine/Initialize(mapload, ...)
```
Comment (stanalbatross) [inline code/modules/clothing/shoes/marine_shoes.dm:24]: should also apply to shoes/veteran (PMC shoes) as well as /obj/item/clothing/shoes/combat and /obj/item/clothing/shoes/souto as they are armoured

**Review by stanalbatross — CHANGES_REQUESTED** [review]

**Comment by TellyLsc** [comment]
Were helmets ever addressed on this PR? Unless they were linked as well to the heavy armor, it would still just shift the target to head wouldn't it?

**Comment by Nanu308** [comment]
I'll put it here as well for reference.

Prior to this being tried out and tested in-game i want Xeno targeting to be changed first in a test, where more RNG is introduced to where they strike.

For example, even if you aim foot you are not always guaranteed to hit the foot as in our current system, but rather a 

1. Weighted RNG system for specific limbs. Such as for example (not final numbers at all.) Chest is 60%, Head is 5%, Each Arm is 10% etc. 

or

2. Evenly RNG spread out across each limb.

**Pull request closed without merging**